src/components/input-date-picker/input-date-picker.stories.ts

```diff
@@ -48,6 +83,24 @@ export const DarkMode = (): string => html`
   </div>
 `;
 
+export const Placement = (): string => html`
+  <div style="width: 400px">
+    <calcite-label layout="inline">
+      Date
+      <calcite-input-date-picker
+        scale="${select("scale", ["s", "m", "l"], "m")}"
+        value="${text("value", "2020-12-12")}"
+        min="${text("min", "2016-08-09")}"
+        max="${text("max", "2023-12-18")}"
+        locale="${select("locale", locales, "en")}"
+        intl-next-month="${text("intl-next-month", "Next month")}"
+        intl-prev-month="${text("intl-prev-month", "Previous month")}"
+        placement="${select("placement", calcite_placements, "auto")}"
```

Comment:
maybe we can import the `defaultMenuPlacement` from the resources for thiscomponent